### ISSUE TITLE
Monit 5.18 has new output format for summary which breaks the module

### ISSUE THREAD

**Issue by mangas** [ansible/ansible-modules-extras#2462]
##### ISSUE TYPE
- Bug Report
##### COMPONENT NAME

monit
##### ANSIBLE VERSION

ansible 1.7.2
##### OS / ENVIRONMENT

Linux - Debian
##### SUMMARY

Monit 5.18 has a new output format incompatible with the current module
##### STEPS TO REPRODUCE

Install Monit
Configure any process

```
ansible localhost -m monit -a "name=zookeeper state=restarted"
```

Get error message '"zookeeper process not presently configured with monit' which is not accurate
##### EXPECTED RESULTS

Should restart the process
##### ACTUAL RESULTS

Get error message '"zookeeper process not presently configured with monit' which is not accurate

**Comment by mangas** [comment]
@nitzmahone any thoughts about https://github.com/ansible/ansible-modules-extras/pull/2463 ?

**Comment by nitzmahone** [comment]
@mangas Sorry, I don't have monit configured anywhere to test that- someone in the community that uses monit will need to do that.

**Comment by mangas** [comment]
This is the package with the right version:
https://packages.debian.org/stretch/monit

Can be tested with debian stretch from docker for example
https://hub.docker.com/_/debian/

Let me know if there is anything I can do to help

**Comment by ansibot** [comment]
@brian-brazil, @dstoflet, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@brian-brazil, @dstoflet, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@brian-brazil, @dstoflet, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by grisha87** [comment]
@brian-brazil , @dstoflet , @mangas This issue is also applicable for newest Ansible (2.1) I was thinking about fixing this myself and going in with a PR but I see fix from @mangas isn't processed yet (PR#2463). Can you chaps take a look?

**Comment by ansibot** [comment]
@brian-brazil, @dstoflet, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@brian-brazil, @dstoflet, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
This repository has been locked. All new issues and pullrequests should be filed in https://github.com/ansible/ansible

Please read through the <a href="http://docs.ansible.com/ansible/dev_guide/repomerge.html#move-issues-and-prs-to-new-repo">repomerge</a> page in the dev guide. The guide contains links to tools which automatically move your issue or pullrequest to the ansible/ansible repo.

**Comment by mangas** [comment]
@grisha87 new PR: https://github.com/ansible/ansible/pull/21891

**Comment by traumschule** [comment]
With Monit 5.25.0 i get the same error:
```
$ ansible localhost -bm monit -a "name=apache state=restarted"
localhost | FAILED! => {
    "changed": false, 
    "failed": true, 
    "msg": "apache process not presently configured with monit", 
    "name": "apache", 
    "state": "restarted"
}
```
